src/js/helpers/useToast.test.ts

```diff
@@ -0,0 +1,122 @@
+import selectorsMap from '@constants/selectors-map';
+import useToast from '@helpers/useToast';
+import { 
+  containerComponent, btnCloseComponent, testMessage, testOptions, defaultType, validTemplate, invalidTemplate 
+} from '@constants/mocks/useToast-data';
+
+describe('useToast', () => {
+  const expectedOptions = {
+    getType(hasType: boolean) {
+
+      return hasType ? `.bg-${testOptions.type}` : `.bg-${defaultType}`;
+    },
+    getClassList() {
```

Comment:
You can define these functions outside the describe so the test is easier to read.

```js
describe
  describe
    it
    it
  describe
    it
```